lib/src/bleed/Bleed.tsx

```diff
@@ -54,8 +54,8 @@ function getSpacingValue(spacingName) {
   }
 }
 
-const StyledBleed = styled.div<BleedProps>`
-  ${({ space, horizontal, vertical, top, right, bottom, left }) => `
+const StyledBleed = styled.div<BleedPropsType>`
+  ${({ space, horizontal, vertical, top, right, bottom, left }: BleedPropsType) => `
```

Comment:
It is not necessary to indicate `BleedPropsType` when passing to styled components.

Reply:
Getting this error if I only indicate `BleedPropsType` in the styled component:

![image](https://user-images.githubusercontent.com/10975289/158986097-aa906f86-a52a-43e4-8bf4-726eb5db4cef.png)

The same for `Row` and `Stack`.

Reply:
I don't have that error. 
![image](https://user-images.githubusercontent.com/35959222/159014734-8b0621f4-1b33-4674-8983-3d4004847ecb.png)

We are using the same approach in the radio group and is working fine...

Reply:
Done!